### PR TITLE
Lenovo port to persistence 1

### PR DESCRIPTION
**Pull request by amuraleedhar**
SUMMARY
CNOS Modules were originally using Paramiko. Here this is an attempt to change from paramiko to persistence connection of Ansible. More over along with this I am taking care of the new CLI changes happend on CNOS. I am adding UT cases for this module also along with

ISSUE TYPE
Bugfix Pull Request

COMPONENT NAME
lib\ansible\modules\network\cnos\cnos_portchannel.py
lib\ansible\modules\network\cnos\cnos_interface.py
lib\ansible\module_utils\network\cnos.cnos.py
test\units\modules\network\cnos\fixtures\cnos_portchannel_config.cfg
test\units\modules\network\cnos\fixtures\cnos_ethernet_config.cfg
test\units\modules\network\cnos\test_cnos_ethernet.py
test\units\modules\network\cnos\test_cnos_portchannel.py

ANSIBLE VERSION
ansible 2.7.0.dev0 (devel f9cbdcd) last updated 2018/07/03 14:55:43 (GMT +550)
config file = /etc/ansible/ansible.cfg
configured module search path = [u'/root/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
ansible python module location = /home/ansible/sheru/ansible/lib/ansible
executable location = /home/ansible/sheru/ansible/bin/ansible
python version = 2.7.6 (default, Nov 23 2017, 15:49:48) [GCC 4.8.4]

ADDITIONAL INFORMATION
After successful review and merging of these module, I will change rest of the modules

### CODE REVIEW

**Comment by amuraleedhar** [comment]
Please merge this PR

File: lib/ansible/modules/network/cnos/cnos_portchannel.py [hunk 1]
```diff
@@ -378,34 +378,34 @@
...
-  password: "{{ hostvars[inventory_hostname]['ansible_ssh_pass'] }}"
-  deviceType: "{{ hostvars[inventory_hostname]['deviceType'] }}"
-  outputfile: "./results/test_portchannel_{{ inventory_hostname }}_output.txt"
-  interfaceRange: 33
-  interfaceArg1: "bfd"
-  interfaceArg2: "interval"
-  interfaceArg3: 55
-  interfaceArg4: 55
-  interfaceArg5: 33
+#- name: Test Port Channel - bfd
```
Comment (Qalthos) [inline lib/ansible/modules/network/cnos/cnos_portchannel.py]: I really don't see the point of comments inside an EXAMPLES string...

**Review by Qalthos — COMMENTED** [review]
Looks like you've taken my comments from the last PR, or at least I don't see anything especially out of place here.

File: lib/ansible/modules/network/cnos/cnos_portchannel.py (hunk 1 above)
Comment (amuraleedhar) [reply lib/ansible/modules/network/cnos/cnos_portchannel.py]: I will remove it now.

**Comment by amuraleedhar** [comment]
Please merge this.

**Pull request merged**